logictest/testdata/aggregate/aggregate_nils

```diff
@@ -0,0 +1,26 @@
+createtable schema=default
```

Comment:
re file name, I would call it `aggregate_nulls` instead of `nils` (even though the previous test mentioned nils) for consistency.